aws-services/uninstall-aws-services.hbs.md

```diff
@@ -0,0 +1,104 @@
+# Uninstall AWS Services
+
+This topic tells you how to uninstall the AWS Services package from the Tanzu Application Platform (commonly
+known as TAP) package repository.
+
+## <a id="prepare-infra"></a> Step 1: Prepare your infrastructure
+
+Before uninstalling the package, you will have to decide what to do with the existing resources you have
+claimed.
+
+* In case you want to keep a resource:
+
+  1. Identify the corresponding Crossplane managed resource:
+
+      ```console
+      kubectl get instances.rds.aws.upbound.io
+      ```
+
+  1. Edit the resource:
+
+      ```console
+      kubectl edit instances.rds.aws.upbound.io <RESOURCE-NAME>
+      ```
+
+  1. Change the resource `deletionPolicy` to `Orphan`:
+
+      ```yaml
+      apiVersion: rds.aws.upbound.io/v1beta1
+      kind: Instance
+      metadata:
+        # ...
+      spec:
+        deletionPolicy: Orphan
+        # ...
+      ```
+
+* In case you want to get rid of a resource:
+
+  1. Delete the corresponding claim:
+
+      ```console
+      tanzu service class-claim delete <CLAIM-NAME>
+      ```
+
+  1. Wait for the resource to disappear in the AWS console.
+
+## <a id="uninstall-package"></a> Step 2: Uninstall the AWS Services package
+
+To uninstall the AWS Services package:
+
+1. Confirm that you have the AWS Services package installed by running:
+
+    ```console
+    tanzu package installed list -A
+    ```
+
+1. Take note of the name and namespace for the package with `aws.services.tanzu.vmware.com` as the
+   `PACKAGE-NAME`.
+1. Uninstall the package:
+
+    ```console
+    tanzu package installed delete <PACKAGE-NAME> -n <PACKAGE-NAMESPACE>
+    ```
+
+    Assuming you have followed our [install instructions](./install-aws-services.hbs.md#install-package), the
+    command will be:
+
+    ```console
+    tanzu package installed delete aws-services -n tap-install
+    ```
+
+## <a id="delete-providerconfig"></a> Step 3: Delete the `ProviderConfig`
+
+You will have to delete the `ProviderConfig` you created after installing the package to allow the
+uninstallation to finalize.
+Unfortunately, the `ProviderConfig` gets a finalizer installed by Crossplane as soon as it's created: in order
+to delete it, we'll have to remove the finalizer first.
+If you don't perform this step, the `ProviderConfig` and its related CRD will hang around in the cluster and
+prevent you from reinstalling the package in the future.
+
+1. Edit the `ProviderConfig`:
+    
+    ```console
+    kubectl edit providerconfig.aws.upbound.io <PROVIDER-CONFIG-NAME>
+    ```
+
+1. Take not of the `Secret` referenced by `spec.credentials`.
+1. Delete the `metadata.finalizers` entry.
```

Comment:
I think you will need to set `metadata.finalizers` to an empty list, not just delete it. If that's the case, lets be more precise.

Reply:
I've always just deleted the whole entry in my tests and it's worked 🤷